bigbang/mailman.py

```diff
@@ -130,7 +130,7 @@ def urls_to_collect(urls_file):
             continue
         if len(url) == 0:   # ignore empty lines
             continue
-        if validators.url(url) != True:
+        if checkers.url(url) != True:
```

Comment:
This fails for me.

`AttributeError: module 'validator_collection.checkers' has no attribute 'url'`

Maybe `checkers.is_url`?